src/shipit_static_analysis/shipit_static_analysis/lint.py

```diff
@@ -69,11 +69,11 @@ def as_text(self):
         '''
         Build the text content for reporters
         '''
-        return '{}: {} [{}: {}]'.format(
+        linter = self.rule and '{}: {}'.format(self.linter, self.rule) or self.linter
```

Comment:
Nit: this is a bit confusing on one line, I would use a more readable if:
```Python
if self.rule:
    linter = ...
else:
    linter = ...
```

Reply:
Or `linter = '{}: {}'.format(self.linter, self.rule) if self.rule else self.linter`